src/main/java/in/projecteka/consentmanager/dataflow/model/hip/DataRequest.java

```diff
@@ -0,0 +1,19 @@
+package in.projecteka.consentmanager.dataflow.model.hip;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
+public class DataRequest {
+    private final String transactionId;
+    private final UUID requestId;
+    private final String timestamp;
```

Comment:
I think we are supposed to use LocalDateTime for timestamp

Reply:
in contract schema, timestamp is string

Reply:
okay cool.. just wanted to confirm

Reply:
I think we should give localDateTime data type for timestamp. Please check with Ganesan once

Reply:
changed to LocalDateTime